### PR TITLE
Simplify theme layout and header logic

### PR DESCRIPTION
**Pull request by Etheryte**
## What does this PR change?

Part one of bringing the Uyuni and Suma theme closer together as part of https://github.com/SUSE/spacewalk/issues/22502.
This PR irons out small inconsistencies, throws out small unused bits, etc.
There's small visual layout shifts in the header, but in general it looks more or less the same.  

Screenshot 2023-09-20 at 11 40 30

## GUI diff

Minor small shifts here and there.

- [x] **DONE**

## Documentation
- No documentation needed: It makes sense to update the documentation only after all the theme changes are done.

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Part of https://github.com/SUSE/spacewalk/issues/22502

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by cbbayburt — APPROVED** [review]
I realized that I'm not up-to-date at all on the stylesheets but I guess at this point none of us are :see_no_evil: 
LGTM as far as I can tell :+1:

**Review by parlt91 — APPROVED** [review]
Looking good to me as well :+1:

**Comment by Etheryte** [comment]
Ping @lucidd since your review is required.

**Review by mcalmer — APPROVED** [review]

**Comment by mcalmer** [comment]
@Etheryte please rebase this PR and let the tests run again. I hope that the errors are solved after this.
At least we cannot see what the problems are as they are too old

File: web/html/src/branding/css/uyuni/legacy-theme.less [hunk 1]
```diff
@@ -18,14 +18,19 @@ input.form-control::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
 }
 
 header {
+  margin: 0 !important;
+  border-radius: 0 !important;
   z-index: 1000;
   width: 100%;
   border: none;
   border-radius: 0;
```
Comment (mcalmer) [inline web/html/src/branding/css/uyuni/legacy-theme.less:26]: 2 times border-radius defined

Comment (Etheryte) [reply web/html/src/branding/css/uyuni/legacy-theme.less:26]: Good catch, thanks.

**Pull request merged**